### PR TITLE
Update gcc.rules

### PR DESCRIPTION
**Pull request by ryanmusante**
GCC the driver program (gcc) calls the compiler (cc1, cc1plus). Should include compilers to rules file

### CODE REVIEW

**Comment by Nefelim4ag** [comment]
Ok, i think then you also need update: rules.broken

File: ananicy.d/00-default/gcc.rules [hunk 1]
```diff
@@ -1 +1,4 @@
+# cc1 and cc1plus high CPU usage compilers invoked by GCC
```
Comment (Nefelim4ag) [inline ananicy.d/00-default/gcc.rules]: I think what compilers with low cpu usage not exists.
I think something like "gcc back-end compilers" more appropriate

Thanks.

Comment (ryanmusante) [reply ananicy.d/00-default/gcc.rules]: You're the programmer, I'm your muse. I trust your recommendation.

**Comment by Nefelim4ag** [comment]
I mean you can remove cc1 from broken, update comment in your patch,
after you can just update PR, instead of closing it =)

**Comment by ryanmusante** [comment]
Oh ok. I will fix this tonight.

**Comment by ryanmusante** [comment]
It appears that cc1 is not being pulled properly so it should stay in rules.broken for now.

**Comment by Nefelim4ag** [comment]
@ryanmusante,
In general cc1 & etc, will be called by gcc/make and must inherit nice & etc.
 - - -
Also, i added skip of young processes, i.e. if process created less then check_freq ago, ananicy ignore them. For don't bother with short lived processes.

**Pull request closed without merging**